services/QuillLMS/app/models/concept_feedback.rb

```diff
@@ -39,7 +39,10 @@ class ConceptFeedback < ApplicationRecord
   def cache_key = "#{ALL_CONCEPT_FEEDBACKS_KEY}_#{activity_type}"
 
   def as_json(options=nil)
-    data
+    translation = translation(locale: "es-la")
```

Comment:
It's possible that this will slow things down a little since we're now pulling from the translation_mappings table each time we grab the as_json on the concept_feedback. Right now those tables are tiny so it shouldn't matter (plus the API call is cached), but I'm open to pulling this data another way from the get-go just in case.

Reply:
I think that your point about the tables being small, but especially the one about us caching the output of this call, means that the extra overhead is totally reasonable.